From: Ralf Habacker <ralf.habacker@freenet.de>
Date: Sun, 23 Aug 2015 19:13:20 +0000
Subject: Fix 'KF5 version can't be launched from the menu' in a frameworks independent way.
X-Git-Url: http://quickgit.kde.org/?p=umbrello.git&a=commitdiff&h=2968078e03c9fa330a97bdeca0219e45d8dad2d8
---
Fix 'KF5 version can't be launched from the menu' in a frameworks independent way.

FIXED-IN:2.17.1 (KDE Applications 15.08.1)
BUG:351621
(cherry picked from commit d31cae83987600e16aa14c48e071a56fcd9c580e)

Conflicts:
umbrello/CMakeLists.txt
---


--- a/umbrello/CMakeLists.txt
+++ b/umbrello/CMakeLists.txt
@@ -556,7 +556,11 @@
         umbrelloui.rc
         DESTINATION ${DATA_INSTALL_DIR}/umbrello )
 else()
-    install( PROGRAMS umbrello.desktop  DESTINATION  ${XDG_APPS_INSTALL_DIR} RENAME org.kde.umbrello.desktop)
+    file(READ umbrello.desktop UMBRELLO_DESKTOP)
+    string(REPLACE "-caption" "-qwindowtitle" UMBRELLO_DESKTOP_OUT "${UMBRELLO_DESKTOP}")
+    file(WRITE ${CMAKE_CURRENT_BINARY_DIR}/umbrello.desktop "${UMBRELLO_DESKTOP_OUT}")
+    install( PROGRAMS ${CMAKE_CURRENT_BINARY_DIR}/umbrello.desktop  DESTINATION  ${XDG_APPS_INSTALL_DIR} RENAME org.kde.umbrello.desktop)
+    install( FILES org.kde.umbrello.appdata.xml  DESTINATION  ${CMAKE_INSTALL_METAINFODIR})
     install( FILES
         umbrelloui.rc
         DESTINATION ${KXMLGUI_INSTALL_DIR}/umbrello )

